packages/core/src/utils/transformers/media/getOptimizedBackgroundImageAsset.spec.ts

```diff
@@ -0,0 +1,88 @@
+import { beforeEach, describe, it, expect } from 'vitest';
+import { getOptimizedBackgroundImageAsset } from './getOptimizedBackgroundImageAsset';
+import { AssetFile } from 'contentful';
+
+let file: AssetFile;
```

Comment:
If you switch this to using `ImageFile` instead you can remove the `!` from `file.details.image!` a few times below

Reply:
Nice, I wasn't aware of `ImageFile`

Reply:
It turns out `ImageFile` in in a private types packages that's used in the web app.  Nevermind this idea